tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -451,6 +462,26 @@ void test_intergal_specs() {
     // Precision
     throw_helper(STR("{:.1}"), integral{0});
 
+    // Locale
+#if !defined(_DLL) || _ITERATOR_DEBUG_LEVEL == DEFAULT_IDL_SETTING
+    assert(format(std::locale{"en-US"}, STR("{:L}"), integral{0}) == STR("0"));

```

Comment:
We're `using namespace std`, so `locale` doesn't need to be `std::`-qualified. Several occurrences.